src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -387,13 +387,7 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $lowercase = true)
                         $invalidBehavior = ContainerInterface::NULL_ON_INVALID_REFERENCE;
                     }
 
-                    if ($strict = $arg->getAttribute('strict')) {
```

Comment:
Well, you kept the attribute in the XSD, which is fine to allow the continuous migration. But you should now trigger the deprecation to allow the removal in 4.0